### PR TITLE
Understood the error and have a YAML ready with firebase and the POM managed with the needed changes

### PR DESCRIPTION
**Pull request by techtronix-868**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (KevinMi2023p) [inline .vscode/settings.json:1]: Do we need a .vscode file for it to work?

**Review by KevinMi2023p — CHANGES_REQUESTED** [review]
I don't think we need the .vscode/settings.json file. Let me know if it's necessary though.

**Comment by techtronix-868** [comment]
Remove it

**Comment by techtronix-868** [comment]
I dont think these should be merged as the other pull request has all the same functionalities.

**Comment by KevinMi2023p** [comment]
Do you mean to remove that file because I can't do that, it has to be the creator. Or do you mean don't merge this PR so that we close it?

**Pull request closed without merging**